.env

```
@@ -1,7 +1,7 @@
-VITE_API_KEY="your_api_key"
-VITE_AUTH_DOMAIN="your domain"
-VITE_PROJECT_ID="your project id"
-VITE_STORAGE_BUCKET="your storage bucket"
-VITE_MESSAGING_SENDER_ID="your sender id"
-VITE_APP_ID="your app id"
-VITE_MEASUREMENT_ID="your measurement id"
+VITE_API_KEY=AIzaSyB7FnQapK68cFbr99GDCD79bveOivdlrpc
+VITE_AUTH_DOMAIN=finlit-b5c1e.firebaseapp.com
+VITE_PROJECT_ID=finlit-b5c1e
+VITE_STORAGE_BUCKET=finlit-b5c1e.firebasestorage.app
+VITE_MESSAGING_SENDER_ID=1047507787657
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_MESSAGING_SENDER_ID key should go before the VITE_PROJECT_ID key

(UnorderedKey)